cmd/run.go

```diff
@@ -221,6 +287,49 @@ func (r *Runner) Run(ctx context.Context, patterns []string) error {
 			return err
 		}
 	}
+	r.config.DirectServer = r.config.Server
```

Comment:
IMO the direct server should be the server and the proxy can be provided for features that know they need it (and the features can create clients to the proxy that know they need to test with the proxy)